### PR TITLE
SCCP-357-Base-Mainnet

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

File: tomls/omnibus-base-mainnet-andromeda/collaterals/synth-cbbtc.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[setting.scbbtc_issuance_ratio]
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/collaterals/synth-cbbtc.toml:1]: we  could compact all these settings into 
```suggestion
[var.scbbtc_synth_config]
```

File: tomls/omnibus-base-mainnet-andromeda/rewards/scbeth_perps_liquidations_rewards.toml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+[provision.RewardsDistributor_scbeth_liquidation_rewards]
+source = "synthetix-rewards-dist-ext:3.6.0"
+target = "synthetix-rewards-dist-ext:3.6.0@perps-scbeth"
+options.salt = "<%= settings.salt %>"
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/rewards/scbeth_perps_liquidations_rewards.toml:4]: for all these we need to update the salt too

```suggestion
options.salt = "<%= settings.salt %>-scbeth"
```

File: tomls/omnibus-base-mainnet-andromeda/rewards/sweth_perps_liquidations_rewards.toml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+[provision.RewardsDistributor_sweth_liquidation_rewards]
+source = "synthetix-rewards-dist-ext:3.6.0"
+target = "synthetix-rewards-dist-ext:3.6.0@perps-sweth"
+options.salt = "<%= settings.salt %>"
+options.rewardManager = "<%= imports.system.contracts.CoreProxy.address %>"
+options.poolId = "<%= settings.sc_pool_id %>"
+options.payoutToken = "<%= extras.synth_weth_token_address %>"
+options.payoutTokenDecimals = "18"
+options.name = "Perps Lido Wrapped ETH Liquidation Rewards"
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/rewards/sweth_perps_liquidations_rewards.toml:9]:
```suggestion
options.name = "Perps Wrapped ETH Liquidation Rewards"
```

File: omnibus-arbitrum-mainnet.toml [hunk 4]
```diff
@@ -297,7 +297,7 @@ args = [
     "<%= settings.account_withdraw_timeout %>",
 ]
 
-[setting.spartan_council_pool_id_setting]
+[setting.sc_pool_id]
```
[the remark this answers is not in the document]
Comment (kaleb-keny) [reply omnibus-arbitrum-mainnet.toml:300]: yeah this will cause issues if we change it? It's not referenced anywhere with `spartan_council_pool_id_setting` ?

[the remark this answers is not in the document]
Comment (kaleb-keny) [reply omnibus-arbitrum-mainnet.toml:300]: also why setting 😅 ?

[the remark this answers is not in the document]
Comment (kaleb-keny) [reply omnibus-arbitrum-mainnet.toml:300]: It's used in collateral configuration, with `sc_pool_id` only

File: tomls/omnibus-base-mainnet-andromeda/collaterals/synth-cbbtc.toml (hunk 1 above)
Comment (kaleb-keny) [reply tomls/omnibus-base-mainnet-andromeda/collaterals/synth-cbbtc.toml:1]: yeah i misunderstood what you meant here, but now i do.... this should not have been committed 😅

Comment (kaleb-keny) [reply tomls/omnibus-base-mainnet-andromeda/collaterals/synth-cbbtc.toml:1]: will fix these on sepolia too

Comment (kaleb-keny) [reply tomls/omnibus-base-mainnet-andromeda/collaterals/synth-cbbtc.toml]: hopefully resolved, please let me know 🙏🏼

File: omnibus-arbitrum-mainnet.toml (hunk 4 above)
[the remark this answers is not in the document]
Comment (noisekit) [reply omnibus-arbitrum-mainnet.toml:300]: this is wrong toml. Not on base, but arb. 
On arb this setting is called spartan_council_pool_id_setting and should not be changed, the build fails because pool creation uses `settings.spartan_council_pool_id_setting`

File: omnibus-base-mainnet-andromeda.toml [hunk 5]
```diff
@@ -220,7 +247,7 @@ defaultValue = "synthetix:3.3.18"
 defaultValue = "op-gas-price-oracle:3.4.0"
 
 [setting.spot_market_package]
-defaultValue = "synthetix-spot-market:3.3.15"
+defaultValue = "synthetix-spot-market:3.6.0"
```
Comment (noisekit) [inline omnibus-base-mainnet-andromeda.toml:250]: should we revert this change?
```suggestion
defaultValue = "synthetix-spot-market:3.3.15"
```

Comment (kaleb-keny) [reply omnibus-base-mainnet-andromeda.toml:250]: helllo sir, db made changes recently to the latest versions and we'll go live to mainnet with those changes

Comment (noisekit) [reply omnibus-base-mainnet-andromeda.toml:250]: No problem. I saw those versions reverted to old ones on testnet and wanted to double-check.

File: omnibus-arbitrum-mainnet.toml (hunk 4 above)
[the remark this answers is not in the document]
Comment (kaleb-keny) [reply omnibus-arbitrum-mainnet.toml:301]: Hoping it's resolved now https://github.com/Synthetixio/synthetix-deployments/pull/506/commits/c4e33fd0f114eee34a4e5f58ae7c254b9be4aa7f

[the remark this answers is not in the document]
Comment (noisekit) [reply omnibus-arbitrum-mainnet.toml:301]: it is not. This change we are talking about here is for arbitrum mainnet toml. Needs to be reverted.

[the remark this answers is not in the document]
Comment (kaleb-keny) [reply omnibus-arbitrum-mainnet.toml]: finally resolved 😅

File: omnibus-arbitrum-mainnet.toml [hunk 6]
```diff
@@ -298,7 +298,7 @@ args = [
     "<%= settings.account_withdraw_timeout %>",
 ]
 
-[setting.spartan_council_pool_id_setting]
+[setting.setting.spartan_council_pool_id_setting]
```
Comment (noisekit) [inline omnibus-arbitrum-mainnet.toml:301]:
```suggestion
[setting.spartan_council_pool_id_setting]
```

**Review by noisekit — APPROVED** [review]
with prettier

**Comment by kaleb-keny** [comment]
I've run all necessary tests @noisekit my end, this PR is ready to be sent, hopefully i'm targeting monday
